test/samples_test.dart

```diff
@@ -13,36 +13,21 @@ const testOptions = const PreprocessorOptions(
     inputFile: 'memory'
 );
 
-void testCSSFile(String name) {
-  var errors = <Message>[];
-  File bootstrap = new File('test/examples/${name}.css');
-  String css = bootstrap.readAsStringSync();
-  var stylesheet = parseCss(css, errors: errors, opts: testOptions);
+void testCSSFile(File cssFile) {
+  final errors = <Message>[];
+  final css = cssFile.readAsStringSync();
+  final stylesheet = parseCss(css, errors: errors, opts: testOptions);
 
-  expect(stylesheet != null, true);
-  expect(errors.isEmpty, true, reason: errors.toString());
+  expect(stylesheet, isNotNull);
+  expect(errors, isEmpty, reason: errors.toString());
 }
 
-testBootstrap() => testCSSFile('bootstrap');
-testFoundation() => testCSSFile('foundation');
-testPure() => testCSSFile('pure');
-testBoilerplate() => testCSSFile('boilerplate');
-testBase() => testCSSFile('base');
-testMDCLayout() => testCSSFile('mdc-layout');
-testMDCCard() => testCSSFile('mdc-card');
-testSkeleton() => testCSSFile('skeleton');
-testBulma() => testCSSFile('bulma');
-testMaterialize() => testCSSFile('materialize');
-
 main() {
-  test('Bootstrap', testBootstrap);
-  test('Foundation', testFoundation);
-  test('Pure', testPure);
-  test('HTML5Boilerplate', testBoilerplate);
-  test('Base', testBase);
-  test('MDC Layout Compiled', testMDCLayout);
-  test('MDC Card', testMDCCard);
-  test('Skeleton', testSkeleton);
-  test('Bulma', testBulma);
-  test('Materialize', testMaterialize);
-}
+  var cssDir = new Directory.fromUri(Platform.script.resolve('./examples'));
+  for (var element in cssDir.listSync())
+    if(element is File && element.uri.pathSegments.last.endsWith('.css')) {
+    
+    File cssFile = element;
```

Comment:
You don't actually need to coerce `element` as a `File` here. The analyzer knows its a file inside this block because it passed the `element is File` condition.